docs/source/installation.rst

```diff
@@ -2,152 +2,133 @@ Installation
 ============
 
 Supported OS
-------------
+++++++++++++
 
-Currently, we only support ``MacOS`` and ``Linux`` operating systems. ``Windows``
-users have the possibility to install and use ``ivadomed`` via
-`Windows Subsystem for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/>`_. The steps below (about updating bashrc) are strongly recommended for MacOS users in the installation process but are optional for Linux users.
+    Currently, ``ivadomed`` supports GPU/CPU on ``Linux`` and ``Windows``, and CPU only on ``macOS`` and `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/>`_.
 
-Open your bash/zsh script file with editor on your computer.
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using bash shell
-    ::
+    You can setup ``ivadomed`` using either Conda or Venv:
 
-        vim ~/.bashrc
+    .. tabs::
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+        .. tab:: Install via ``venv``
 
-Write in your .bashrc/.zshrc file with following line.
+            1. Setup Python Venv Virtual Environment.
 
-::
+                ``ivadomed`` requires Python >= 3.6 and <3.10.
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+                First, make sure that a compatible version of Python 3 is installed on your system by running:
 
-Save this change and restart your terminal to apply the change.
+                .. code::
 
-There are fundamentally two different approaches to install ``IvadoMed``:
+                   python3 --version
 
-1) Install via Conda
-    This is the easiest way for personal computers.
+                If your system's Python is not 3.7, 3.8, or 3.9 (or if you don't have Python 3 installed at all), please `install Python <https://realpython.com/installing-python/>`_ before continuing.
 
-2) Install via Venv/VirtualEnv
-    This is compatible with ComputeCanada cluster environment.
+                Once you have a supported version of Python installed, run the following command (replacing ``3.X`` with the Python version number that you installed):
 
-Approach 1: Conda
-------------------
+                .. code::
 
-Step 1: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+                   python3.X -m venv ivadomed_env
```

Comment:
This works great now.
I wanted to mention that the first time I tried this, I had to install python3.8-venv separately (I already had python 3.8 without venv apparently), see this https://github.com/ivadomed/ivadomed/pull/969#discussion_r744296259.

So I don't know if that is normal or not vut a user may have to do the same (I don't know how to test this now that my python3.7-venv is already installed)

Reply:
> (I already had python 3.8 without venv apparently)

Huh. It was my impression that `venv` came [bundled with Python](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#installing-virtualenv):

> Note: If you are using Python 3.3 or newer, the venv module is the preferred way to create and manage virtual environments. venv is included in the Python standard library and requires no additional installation.

But, as you've pointed out, `python-venv` needs to be installed separately on Debian/Ubuntu systems, or else the following error is given:

```
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt install python3.8-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.
```

Yikes. I didn't know that, so thank you for pointing it out. (I missed it the first time you pointed it out, oops.) :sweat:

Reply:
Apparently this is a bug in Debian/Ubuntu specifically: https://bugs.launchpad.net/ubuntu/+source/python3.4/+bug/1290847

We might benefit from adding an additional message in this section that says:

> If you use Debian or Ubuntu, make sure to install the `python3-venv` module alongside whichever version of Python you choose to install. For other operating systems, `venv` will be installed by default.

But, at the same time, the error message given by Python is quite clear, and provides a command for the user to run, so perhaps we can just leave it be?

Reply:
I like the additional message to help inexperienced users, but I would also be OK to leave it be because the python message is indeed very clear 🙂
In any case, this is minor and I don't think it should block the merge of the PR.

Reply:
Let's be a bit more verbose. Tab already hide tons of parallel path already for those not interested so our main path is actually already much shorter than others. Better safe than sorry. Thanks for testing this out!